### PR TITLE
Paypal-Payment 

### LINKED ISSUE #206: Payment mockup
### User Story
**Who:** As a developer
**What:** I want to be able to mock users payment
**Why:** to be able to test order payment 

### Acceptance Criteria:
After clicking Payment button on Order review Page user will be redirected to a window with payment system. See below example:
image

### Implementation Details:
Payment system testing can be done with [Paypal ](https://developer.paypal.com/docs/integration/direct/payments/test-the-api/) or [Stripe](https://stripe.com/docs/testing).
The payment should proceed without actually charging money.

### PR DESCRIPTION
**Pull request by santhoshboinapally**
# Description
Creating a Paypal Component. When User click Proceed to Payment Button in Shopping cart it will redirected to Paypal Website to finish payments. **Created Dummy Paypal Page and path i will remove after review** 
Please provide a short summary explaining what this PR is about.

Fixes #206 

# How to test?
npm run dev
go to  " http://127.0.0.1:3000/paypal " to check paypal.

Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/App.js [hunk 1]
```diff
@@ -57,7 +58,10 @@ function App() {
         <Route exact path="/order-confirmation">
           <OrderConfirmationPage />
         </Route>
-
+        {/* Testing Paypal page path,should be removed  */}
+        <Route exact path="/paypal">
+          <PaypalPage />
+        </Route>
```
Comment (Divyajg) [inline src/client/App.js]: Paypal path should go to authenticate route!

File: src/client/components/Paypal/Paypal.component.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            onSuccess(details);
+          });
+        },
+        onError: (error) => {
+          onError(error);
+        },
+      })
+      .render(paypalRef.current);
+  }, [orderId, toalAmount, userName, onSuccess, onError]);
+  return <div ref={paypalRef} />;
```
Comment (Divyajg) [inline src/client/components/Paypal/Paypal.component.js:43]: You can create a success pop window after successful payment. here for now!

**Review by Divyajg — COMMENTED** [review]
Storybook is looks fine, but complaining something about button. I couidn't figure that out!

**Review by FlorentinaPetica — APPROVED** [review]

File: src/client/App.js (hunk 1 above)
Comment (santhoshboinapally) [reply src/client/App.js]: I remove this path, its only for testing the component.

**Review by pankaj28843 — APPROVED** [review]

**Pull request merged**